openmc/mgxs/library.py

```diff
@@ -350,7 +350,7 @@ def build_library(self):
     def add_to_tallies_file(self, tallies_file, merge=True):
         """Add all tallies from all MGXS objects to a tallies file.
 
-        NOTE: This assumes that build_library() has been called
+        NOTE: This assumes that :meth:`Library.build_library` has been called
```

Comment:
Oh! Well in that case, I see a number of places where you use `:math:` instead...perhaps you meant to use `:meth:`?